### ISSUE TITLE
CopyFolderComponent: parempi exclude

### ISSUE THREAD

**Issue by saviit** [TIM-JYU/TIM#3672]
`CopyFolderComponent`in exclude hyväksyy tällä hetkellä ainoastaan validin Python-regexpin, mikä hankaloittaa erityisesti tavallisen käyttäjän elämää, jos hän haluaa helposti määritellä kopioinnin ulkopuolelle jäävät dokumentit/alakansiot.

Intuitiivisempi lähestymistapa olisi tehdä exlude esim. seuraavasti
- excluden-tekstikentän yhteyteen checkbox, jolla valitaan regexpin ja vapaan tekstin välillä
- excludettavat voi antaa listana, erottimena pilkku/puolipilkku, kun valitsee vapaan tekstivalinnan (edellinen kohta)
- vapaatekstinä syötetystä tekstistä parsitut nimet muunnetaan Pythonin puolella muotoon `.*teksti.*`
  -  riittää siis antaa vain osa nimestä, ei tarvitse olla täsmällinen osuma

Lisäksi pitää lisätä linkki ohjeisiin, missä kerrotaan miten tuon exluden pitäisi toimia, minimissään linkki Pythonin omaan regexp-ohjeeseen.

**Comment by vesal** [comment]
paras olisi jos tulisi lista kaikesta kopoitavista siinä pr3view ja ruksi viereen joita voi ottaa pois tarpeen mukaan ja nopeuttaa tuota sillä RegExp filtterillä

**Comment by saviit** [comment]
> paras olisi jos tulisi lista kaikesta kopoitavista siinä pr3view ja ruksi viereen joita voi ottaa pois tarpeen mukaan ja nopeuttaa tuota sillä RegExp filtterillä

Joo, niinhän se olisi, mutta siinä on jo vähän enemmän hommaa (pitää mm. muokata sitä miten se preview ylipäätään toimii), tuon mun ehdotuksen saa hyvin pienellä vaivalla näperrettyä.
